packages/mobile/src/hooks/useLoadingAnimation.ts

```diff
@@ -0,0 +1,17 @@
+import { useRef, useLayoutEffect } from 'react'
```

Comment:
For hooks that we're unlikely to reuse, best to just include it in the single component that uses it imo rather than a separate file!